metrics/helpers/lp.py

```diff
@@ -123,6 +123,52 @@ def get_team_daily_triage_count(team, distro, blacklist=None):
     return len(results)
 
 
+def get_team_unassigned_bugs(team, distro):
+    """Report count of new unassigned bugs for Launchpad team on a distro."""
```

Comment:
I think it'd be good to clarify that this function also filters on the bug subscriber.